server/src/common/constants/cors.constants.ts

```diff
@@ -0,0 +1,9 @@
+import { ENV } from '../enum';
+
+const LOCAL_HOST = `http://localhost:${ENV.APP.PORT}/`;
```

Comment:
Do we need port here?

Reply:
If we change the port in .env, we will need to manually change here.